client/app/lib/helpers/url-helpers.js

```diff
@@ -21,5 +21,25 @@ function getUrlParameter(sParam) {
   return '';
 }
 
+/**
+ * Get the course id from URL.
+ *
+ * return {number}
+ */
+function getCourseId() {
+  const match = window.location.pathname.match(/^\/courses\/(\d+)/);
+  return match && match[1];
+}
+
+/**
+ * Get the survey id from URL.
+ *
+ * return {number}
+ */
+function getSurveyId() {
```

Comment:
I think current form is better in readability and performance ( this method will be used quite often ), since the method is short, maybe we keep it as it is ?